client/src/sim.rs

```diff
@@ -279,6 +290,20 @@ impl Sim {
         }
     }
 
+    fn update_blinker_state(&mut self, id: EntityId, new_blinker: &Blinker) {
+        match self.entity_ids.get(&id) {
+            None => debug!(%id, "blinker state update for unknown entity"),
+            Some(&entity) => match self.world.get::<&mut Blinker>(entity) {
+                Ok(mut blinker) => {
+                    blinker.on = new_blinker.on;
```

Comment:
This can be changed to
```rust
*blinker = new_blinker.clone();
```
to avoid potentially forgetting to update this if adding another field to `Blinker`.

Reply:
That's a good idea. Done.